app/views/checkout/payment.scala.html

```diff
@@ -1,16 +1,22 @@
-@(form: Form[model.SubscriptionData])
+@(form: Form[model.SubscriptionData], userIsSignedIn: Boolean)
 
-@import configuration.Config
+@import configuration.Config.Identity._
+@import routes.Checkout.renderCheckout
 
 @main("Checkout | Subscriptions | The Guardian") {
 
+
 <div class="gs-container">
     @fragments.page.header("Checkout")
 
-
-    <div class="page-section__content">
+    <div class="page-section__content" data-user-signedin="@userIsSignedIn">
```

Comment:
Not sure I follow the logic here. Not a massive fan of using the DOM to model data on single div on a single template. Can you expose this as a config object globally on uncached pages?